lib/ansible/modules/apt_repository.py

```diff
@@ -24,22 +24,26 @@
     repo:
         description:
             - A source string for the repository.
+        type: str
         required: true
     state:
         description:
             - A source string state.
+        type: str
         choices: [ absent, present ]
         default: "present"
     mode:
         description:
-            - The octal mode for newly created files in sources.list.d
-        default: '0644'
+            - The octal mode for newly created files in sources.list.d.
+            - Default is what system uses (probably 0644).
```

Comment:
better remove ' (probably 0644)'

Reply:
This has been discussed on https://github.com/ansible/ansible/pull/70319#discussion_r448980070 too.

But now that you mentioned this, I haven't seen this line before:
https://github.com/ansible/ansible/blob/db519bc17c1ff0c6e095d022bfe988eb87630090/lib/ansible/modules/apt_repository.py#L143
https://github.com/ansible/ansible/blob/db519bc17c1ff0c6e095d022bfe988eb87630090/lib/ansible/modules/apt_repository.py#L322

It seems that default `mode` is `0644` and maybe that was why it has been mentioned on `mode` doc, am I right?

If so I think adding `default: '0644'` on doc would be appropriate because it explains the module behaviour as it is.

Reply:
Also this: https://github.com/ansible/ansible/commit/127b854d2c0eda8de6954cc5eac46e6681577c65

Reply:
@felixfontein Thanks, I didn't know this. What's your opinion on removing `(probably 0644)`?
It can be helpful for those who care about their `apt_repository` file permission, also it gives a hint that they can use setuid, setgid and sticky bits and they should put a leading zero if they don't want them, so maybe changing it to `(for example 0644)` can be better than removing this.

Reply:
I think it is ok to leave it for now. I guess the core team needs to decide whether they want to fix the default to be `0644` (that would require a behavior change), or remove it completely.